### PR TITLE
[NOTEPAD] Russian (ru-RU) translation update

### PR DESCRIPTION
**Pull request by Artem13327**
Google Translate helped me to translate some lines

### CODE REVIEW

**Comment by JoachimHenze** [comment]
cc from chat:
@Artem13327 hi, there are two problems with PR #5181 (notepad)
    line 28 - reusing mnemonic key "&о" which is already used in CMD_OPEN
    line 179 - STRING_NOWPRINTING translation is not correct

File: base/applications/notepad/lang/ru-RU.rc [hunk 1]
```diff
@@ -168,19 +168,19 @@ BEGIN
...
     STRING_PRINTERROR "Невозможно распечатать файл '%s'.\n\nПроверьте, включен ли принтер, и настроен ли он правильно."
     STRING_DEFAULTFONT "Lucida Console"
-    STRING_LINE_NUMBER_OUT_OF_RANGE "The specified line number is out of range."
-    STRING_NOWPRINTING "Now printing page..."
-    STRING_PRINTCANCELING "The print job is being canceled..."
-    STRING_PRINTCOMPLETE "Printing is successfully done."
-    STRING_PRINTCANCELED "Printing has been canceled."
-    STRING_PRINTFAILED "Printing is failed."
+    STRING_LINE_NUMBER_OUT_OF_RANGE "Указанный номер строки находится вне допустимого диапазона."
+    STRING_NOWPRINTING "Печетаемая страница..."
```
Comment (SigmaTel71) [inline base/applications/notepad/lang/ru-RU.rc:179]:
```suggestion
    STRING_NOWPRINTING "Идёт печать страницы..."
```

File: base/applications/notepad/lang/ru-RU.rc [hunk 2]
```diff
@@ -168,19 +168,19 @@ BEGIN
...
     STRING_LINE_COLUMN "Строка %d, столбец %d"
     STRING_PRINTERROR "Невозможно распечатать файл '%s'.\n\nПроверьте, включен ли принтер, и настроен ли он правильно."
     STRING_DEFAULTFONT "Lucida Console"
-    STRING_LINE_NUMBER_OUT_OF_RANGE "The specified line number is out of range."
-    STRING_NOWPRINTING "Now printing page..."
-    STRING_PRINTCANCELING "The print job is being canceled..."
-    STRING_PRINTCOMPLETE "Printing is successfully done."
-    STRING_PRINTCANCELED "Printing has been canceled."
-    STRING_PRINTFAILED "Printing is failed."
+    STRING_LINE_NUMBER_OUT_OF_RANGE "Указанный номер строки находится вне допустимого диапазона."
```
Comment (SigmaTel71) [inline base/applications/notepad/lang/ru-RU.rc:178]:
```suggestion
    STRING_LINE_NUMBER_OUT_OF_RANGE "Указанный номер строки превышает число строк в файле."
```

Comment (katahiromz) [reply base/applications/notepad/lang/ru-RU.rc]: The negative or zero line number is not acceptable.

File: base/applications/notepad/lang/ru-RU.rc [hunk 3]
```diff
@@ -168,19 +168,19 @@ BEGIN
...
     STRING_LINE_COLUMN "Строка %d, столбец %d"
     STRING_PRINTERROR "Невозможно распечатать файл '%s'.\n\nПроверьте, включен ли принтер, и настроен ли он правильно."
     STRING_DEFAULTFONT "Lucida Console"
-    STRING_LINE_NUMBER_OUT_OF_RANGE "The specified line number is out of range."
-    STRING_NOWPRINTING "Now printing page..."
-    STRING_PRINTCANCELING "The print job is being canceled..."
-    STRING_PRINTCOMPLETE "Printing is successfully done."
-    STRING_PRINTCANCELED "Printing has been canceled."
-    STRING_PRINTFAILED "Printing is failed."
+    STRING_LINE_NUMBER_OUT_OF_RANGE "Указанный номер строки превышает число строк в файле."
```
Comment (katahiromz) [inline base/applications/notepad/lang/ru-RU.rc:178]: The negative or zero line number is not acceptable. Forget Windows.

**Review by katahiromz — CHANGES_REQUESTED** [review]

File: base/applications/notepad/lang/ru-RU.rc (hunk 2 above)
Comment (katahiromz) [reply base/applications/notepad/lang/ru-RU.rc]: Could you think the best translation?

Comment (katahiromz) [reply base/applications/notepad/lang/ru-RU.rc]: The user might wrongly input the line number of zero, negative, or beyond the maximum line number.

Comment (SigmaTel71) [reply base/applications/notepad/lang/ru-RU.rc]: > What do you think about this?

Looks acceptable enough to me.

Comment (SigmaTel71) [reply base/applications/notepad/lang/ru-RU.rc]: > Указанный номер строки находится вне допустимого диапазона.

It looks long to me, that's why I put a suggestion, and Katayama evolved it into «Указанный номер строки вне допустимого диапазона», without «находится» which is easily caught in context of what user is doing.
> Указанная строка находится вне допустимого диапазона.

Doesn't make any sense regarding the context (you input the line number, but not the line).

File: base/applications/notepad/lang/ru-RU.rc (hunk 3 above)
Comment (SigmaTel71) [inline base/applications/notepad/lang/ru-RU.rc:178]:
```suggestion
    STRING_LINE_NUMBER_OUT_OF_RANGE "Указанный номер строки вне допустимого диапазона."
```
> Please submit this as Suggested change

Is that your final answer? ©

**Review by katahiromz — APPROVED** [review]
Go.

**Review by SigmaTel71 — APPROVED** [review]

**Pull request merged**